src/worker.ts

```diff
@@ -1,32 +1,25 @@
 import "unfetch/polyfill";
 import { CONFIG_URL } from "./constants";
+import chooseRandomTasks from "./lib/chooseRandomTasks";
 import retry from "./util/promiseRetry";
-import Fetch from "./tasks/fetch";
-import Pop from "./tasks/pop";
+import sequence from "./util/promiseSequence";
+import { create as createTask } from "./tasks";
```

Comment:
AS per my other comment about module naming convention for Go. I prefer it to be `tasks.create` instead of `tasks.createTask`. https://github.com/golang/go/wiki/CodeReviewComments#package-names

Reply:
Okay. Not sure all these Go conventions apply in a JavaScript world, especially when it comes to import-renaming. Go is not JavaScript, and different languages often have different idioms for good reasons.

In the Go example, they are saying that referring to `chubby.chubbyFile` is needlessly redundant, and that makes sense. But you're not dealing with `task.create` in this file, which retains the namespace. Instead you're renaming `create` to `createTask` in order to retain context that was lost by following these Go conventions. But renaming imports can cause a clear disconnect because it don't believe it will be obvious to future maintainers that `createTask` is the same as `task.create`.  

Having said that, I don't feel particularly strongly about it, and it sounds like a convention you'd like to continue to pursue. Will be curious to see how it plays out over the lifetime of the codebase.